client/containers/DashboardOverview/PubOverview/PubOverview.tsx

```diff
@@ -2,19 +2,20 @@ import React from 'react';
 import dateFormat from 'dateformat';
 import { Menu, MenuItem, Tag } from '@blueprintjs/core';
 
-import { usePageContext } from 'utils/hooks';
-import { getPubPublishedDate, getPubLatestReleaseDate } from 'utils/pub/pubDates';
+import { ContributorsList, DashboardFrame, PubHeaderBackground } from 'components';
+import { DefinitelyHas, PubPageData } from 'types';
 import { formatDate } from 'utils/dates';
+import { getPubPublishedDate, getPubLatestReleaseDate } from 'utils/pub/pubDates';
+import { usePageContext } from 'utils/hooks';
 import CitationsPreview from 'containers/Pub/PubHeader/CitationsPreview';
-import { ContributorsList, DashboardFrame, PubHeaderBackground } from 'components';
 
 import { getAllPubContributors } from 'utils/contributors';
 import PubTimeline from './PubTimeline';
 
 require('./pubOverview.scss');
 
 type Props = {
-	pubData: any;
+	pubData: DefinitelyHas<PubPageData, 'reviews'>;
```

Comment:
The `PubPageData` type is mostly (exclusively?) used on the Pub document page, and that's not what we're getting here. It's like a `SanitizedPubData` with some extra citation bits...you could reference that type here if you move it out of `server/`, or you could just define a new one inline here.